my-app/src/App.js

```diff
@@ -1,10 +1,31 @@
-import React from 'react';
+import React, {useState, useEffect} from 'react';
 import './App.css';
 
 function App() {
+
+   // const [user, setUser] = useState({});
```

Comment:
Yes that is the next step that the front end team will do, try to get a page where we can type in these values instead of having it in the file itself

Reply:
But they probably don't need it to be included as a comment, so it can be removed for now